db/migrate/20160617123557_add_meta_fields_to_theming.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetaFieldsToTheming < ActiveRecord::Migration
+  def change
+    add_column :pageflow_themings, :default_author, :string
+    add_column :pageflow_themings, :default_publisher, :string
+    add_column :pageflow_themings, :default_keywords, :string
```

Comment:
These [placeholders are now in place](https://github.com/scrollytelling/pageflow/commit/37d51c4d29b78ee6c65b0f66d5a96ab7c4c005d4).